package.json

```diff
@@ -68,8 +68,30 @@
         "command": "peacock.changeColorToFavorite",
         "title": "Change to a Favorite Color",
         "category": "Peacock"
+      },
+      {
+        "command": "peacock.changeColorOfLiveShareHost",
+        "title": "Change Color for Live Share Host",
+        "category": "Peacock"
+      },
+      {
+        "command": "peacock.changeColorOfLiveShareGuest",
+        "title": "Change Color for Live Share Guest",
+        "category": "Peacock"
       }
     ],
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "peacock.changeColorOfLiveShareHost",
+          "when": "peacock:liveshare == true"
```

Comment:
This will enable the command palette commands `only when Live Share extension is installed` by the user.